app/assets/themes/fearlesscities/_footer.scss

```diff
@@ -0,0 +1,6 @@
+@import "variables";
+
+.mini-footer, .main-footer {
+  //background-image: image-url("fearlesscities/texture-blue.jpg");
```

Comment:
```suggestion
  background-image: image-url("fearlesscities/texture-blue.jpg");
```

Reply:
I this it's better to remove that line, than using it. 

![image](https://user-images.githubusercontent.com/41389482/122731664-cef67200-d27b-11eb-8fc9-2d9b8ad2cd87.png)

Since there are two footers, we should rewrite both of them to work as only one, and then use this background image, but i think that it's better to only use the colour.

Reply:
You could make a fix using a `::before` pseudo-element:

![image](https://user-images.githubusercontent.com/8806781/122756944-569daa00-d297-11eb-8048-4d814a7a630b.png)


```scss
@import "variables";

.mini-footer,
.main-footer {
  background-color: transparent;
  display: block;
  z-index: 1;
}

.main-footer::before {
  content: "";
  height: 100vh;
  width: 100vw;
  display: block;
  position: absolute;
  z-index: -1;
  top: 0;
  background-image: image-url("fearlesscities/texture-blue.jpg");
  background-repeat: no-repeat;
  background-position: bottom;
}
```

Reply:
bought!